### ISSUE TITLE
Can't run predictions on NOCS

### ISSUE THREAD

**Issue by kausiksivakumar** [wenbowen123/BundleTrack#16]
Running command 
`python scripts/run_nocs.py --nocs_dir [PATH_TO_NOCS] --scene_id 1 --port 5555 --model_name can_arizona_tea_norm`
results in the following error
![image](https://user-images.githubusercontent.com/89941846/148125029-d67c473b-63c6-4e39-8d52-df3655e16563.png)
I couldn't find these files in the NOCS real_test directory. Could you share the same
Any idea how to go about this
Thanks

**Comment by wenbowen123** [comment]
Did you download the NOCSdataset as the first step in "Run predictions on NOCS" in readme?

**Comment by kausiksivakumar** [comment]
Yes I did, they only contained these files 
![image](https://user-images.githubusercontent.com/89941846/148276865-823e6bb1-37cb-449e-955d-9b0e3b08b034.png)
the file `real_test/can_arizona_tea_norm.obj` is not present

**Comment by wenbowen123** [comment]
Thanks for reporting this! Please check the updated readme and let me know if there is any problem.

**Comment by kausiksivakumar** [comment]
Yes, I have updated it. I was able to find 'can_arizona_tea_norm.obj'. But the .mtl files were missing in the same. I am attaching the screenshot here
![image](https://user-images.githubusercontent.com/89941846/148607838-2ddea2b5-82c9-40fd-8e12-2dadaa392a7a.png)
Not sure how to go about this, thanks for helping

**Comment by wenbowen123** [comment]
Good to see it works. The .mtl file should not affect running. Feel free to ignore it.

**Comment by kausiksivakumar** [comment]
Thanks, I tried going to the location where the results would be present `/tmp/BundleTrack`. After running the script `run_nocs.py` as specified in the instructions. I found that the file itself is empty. Is this okay? 
![image](https://user-images.githubusercontent.com/89941846/149020670-fa02492a-b0bf-4858-bf49-258aef9d1e01.png)
I thought that we would be getting the 6D poses of the objects whose masks were provided in the results. Did I get it wrong? Could someone help with this, thanks
